src/pages/320x240x16/advanced/mixer_limits.c

```diff
@@ -30,27 +30,27 @@ static void _show_limits()
     int y = ROW1;
     int height = 20;
     //Row 1
-    GUI_CreateLabelBox(&gui->reverselbl, COL1, y, LABEL_WIDTH, 0, &LABEL_FONT, NULL, NULL, _tr("Reverse"));
+    GUI_CreateLabelBox(&gui->reverselbl, COL1, y, LABEL_WIDTH, 0, &LABEL_FONT, GUI_Localize, NULL, _tr_noop("Reverse"));
     GUI_CreateTextSelect(&gui->reverse, COL2, y, TEXTSELECT_96, toggle_reverse_cb, reverse_cb, (void *)((long)mp->channel));
     y += height;
     //Row 2
```

Comment:
Lint Error:
```
32:  Should have a space between // and comment  
34:  Use int16/int64/etc, rather than the C type long  
36:  Should have a space between // and comment  
```